client/overview/index.js

```diff
@@ -144,27 +149,83 @@ const OverviewPage = () => {
 					) }
 				</Notice>
 			) }
-			<TestModeNotice
-				currentPage="overview"
-				isDevMode={ isDevMode }
-				actions={
-					isDevMode
-						? [
-								{
-									label: strings.notice.actions.setUpPayments,
-									onClick: () =>
-										setLivePaymentsModalVisible( true ),
-								},
-								{
-									label: strings.notice.actions.learnMore,
-									url:
-										'https://woocommerce.com/document/woopayments/testing-and-troubleshooting/sandbox-mode/',
-									urlTarget: '_blank',
-								},
-						  ]
-						: []
-				}
-			/>
+			{ isDevMode ? (
+				<BannerNotice status="warning" isDismissible={ false }>
+					{ interpolateComponents( {
```

Comment:
Interesting. I have never seen the `interpolateComponents` technique. It's like reading Flutter/Dart's declarative code 👍